tests/healpix_test.py

```diff
@@ -48,6 +48,14 @@ def test_vec2px(self):
         px = self.hpb.crd2px(x,y,z)
         self.assertEqual(len(px), len(x))
         n.testing.assert_allclose(px, n.array([3728, 2192, 1069,  247,   19,   13,  225, 1023, 2128, 3664]))
+    def test_px2vec(self):
+        x,y = n.linspace(-.5,.5,10), n.linspace(-.5,.5,10)
+        z = 1 - n.sqrt(x**2 + y**2)
```

Comment:
Um, do these assignments to x, y, and z do anything? Aren't they immediately overwritten in line 55?

Reply:
Yes.  Just sloppiness on my part.  Sorry.